tools/testsys-config/Cargo.toml

```diff
@@ -7,12 +7,12 @@ edition = "2021"
 publish = false
 
 [dependencies]
-bottlerocket-types = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", version = "0.0.8", tag = "v0.0.8"}
+bottlerocket-types = { git = "https://github.com/bottlerocket-os/bottlerocket-test-system", version = "0.0.8", branch = "develop"}
```

Comment:
Are we sure this won't cause issues? If 2 commits have the same rust versions which commit is used?

Reply:
This is temporary until we make a TestSys release.